### ISSUE TITLE
Reproducible research - why using RStudio projects and Notebooks is a good idea

### ISSUE THREAD

**Issue by MikeKSmith** [mexindian/R-Users-Canterbury#2]
Hi,

I've been using RStudio projects and Markdown Notebooks for a little while now and I'd be happy to lead a demo / tutorial on using these to ensure that your projects are tidy, well maintained and reproducible i.e. you can come back to them at any point in the future to repeat your analysis. Alternatively, you can use them to create a "pipeline" for your analysis & report that means if the data changes you can quickly reproduce the report reflecting the latest set of data... VERY handy.

Mike

**Comment by DataStrategist** [comment]
Hrm... the community in catnerbury is very much a beginner community, although it seems that there are some hidden R-Users that still were not made aware that might be joining... but just wondering... could you make this presentation even wider, to target excel users and talk about reproducible research from a data analysis perspective and then slowly grow them into R and then as a final touch to talk about RStudio Projects?

**Comment by MikeKSmith** [comment]
Sure, I can tailor it towards Excel users. In fact I see R and Notebooks / reproducibility as something important to teach that community.  I'll mock up an example and put the content here on GitHub so you can provide input and comment as it evolves.

**Comment by DataStrategist** [comment]
How you comin @MikeKSmith ? Do you think a mid-late Jan presentation is feasible?

**Comment by MikeKSmith** [comment]
Hi Amit!

Working on it... First draft of something here: https://github.com/MikeKSmith/R-Users-Canterbury/tree/master/tidy_working_with_R

mid-January should be fine.

Let me know what you think of the content, whether it's pitched right etc. 

Will do a pull-request after the event to merge back in.

Mike

**Comment by DataStrategist** [comment]
Yeah man, that looks perfect. Thanks! Also... actually, I have a completely separate matter to discuss with you... can you hit me up at my work email? akohli at acdivoca.org

**Comment by MikeKSmith** [comment]
Sorry I've not followed up on this recently, been crazy busy at work. Can we pencil in a date avoiding Wednesday or Thursday nights...?

**Comment by DataStrategist** [comment]
Wow... has it been so long? Sorry man, one thing led to another. You still interested in this? Let's plan it! Shall we discuss by phone?

**Comment by MikeKSmith** [comment]
Hi there. Yeah, sorry it has been so long.  Doing this sometime in May would be good. I'll try to pick out some reasonable dates and let you know.

**Comment by DataStrategist** [comment]
Sounds good. Please do give me some good warning time so I can try to get
things organized as much as possible. Thanks

**Comment by MikeKSmith** [comment]
So May has become June, really. How about Wednesday 13th?

**Comment by DataStrategist** [comment]
Sure! I'm asking about the localle. BTW... we were talking about
transferring the group over to someone?

**Comment by DataStrategist** [comment]
Hi @MikeKSmith Nogo on the free place I had in mind... do you know anywhere we could use? Otherwise we'll have to rent somewhere
